src/GalleryTools/Commands/ReserveNamespacesCommand.cs

```diff
@@ -0,0 +1,265 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Autofac;
+using Microsoft.Extensions.CommandLineUtils;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+using NuGet.Services.Entities;
+using NuGet.Versioning;
+using NuGetGallery;
+
+namespace GalleryTools.Commands
+{
+    public static class ReserveNamespacesCommand
+    {
+        private const int DefaultSleep = 0;
+        private const string PathOption = "--path";
+
+        public static void Configure(CommandLineApplication config)
+        {
+            config.Description = "Bulk reserve namespaces";
+            config.HelpOption("-? | -h | --help");
+
+            var pathOptions = config.Option(
+                $"-p | {PathOption}",
+                "A path to a simple text file, which is the list of namespaces to reserve. One namespace per line. Prefix namespaces should end with '*', otherwise they will be exact match reservations.",
+                CommandOptionType.SingleValue);
+
+            var sleepDurationOption = config.Option(
+                "-s | --sleep",
+                $"The duration in seconds to sleep between each reservation (default: 0).",
```

Comment:
Should this be referring to the constant?

```suggestion
                $"The duration in seconds to sleep between each reservation (default: {DefaultSleep}).",
```